mmv1/third_party/terraform/services/compute/compute_instance_helpers.go.erb

```diff
@@ -397,6 +400,19 @@ func expandIpv6AccessConfigs(configs []interface{}) []*compute.AccessConfig {
 			if ptr, ok := data["public_ptr_domain_name"]; ok && ptr != "" {
 				iacs[i].PublicPtrDomainName = ptr.(string)
 			}
+			if eip, ok := data["external_ipv6"]; ok && eip != "" {
+				iacs[i].ExternalIpv6 = eip.(string)
+			}
+			if eipl, ok := data["external_ipv6_prefix_length"]; ok && eipl != "" {
```

Comment:
Note: This won't break instance template even on a default change API-side because the field was already present on the resource